apps/calendar/js/controllers/notifications.js

```diff
@@ -55,26 +55,27 @@ function issueNotification(alarm) {
     }
 
     var begins = calc.dateFromTransport(busytime.start);
-    var distance = dateFormat.fromNow(begins);
-    var now = new Date();
+    return dateFormat.relativeDate(begins).then(distance => {
```

Comment:
We're trying to mostly use [co](https://github.com/tj/co) in calendar for flow control around promises while we wait for async/await